### PR TITLE
remove ubuntu1404 support; fixed few things

### PR DESCRIPTION
**Pull request by mgrainer**

### CODE REVIEW

File: test/fixtures/cookbooks/atlantis_test/recipes/server.rb [hunk 1]
```diff
@@ -29,22 +29,17 @@
...
   version '0.8.3'
-  #checksum 'ca3d237a75f76e08cf4d8a33eba8aaceefeee8a21c4bc81ed971e88350e372b5'
+  checksum '8525d3e88b35e170547512f5b7d5123494d0051f76641e253c979b07629efe08'
   action :install
 end
 
 terraform_installer 'terraform' do
   version '0.12.5'
-  #checksum 'babb4a30b399fb6fc87a6aa7435371721310c2e2102a95a763ef2c979ab06ce'
+  checksum 'babb4a30b399fb6fc87a6aa7435371721310c2e2102a95a763ef2c979ab06ce2'
```
Comment (mgrainer) [inline test/fixtures/cookbooks/atlantis_test/recipes/server.rb]: @ehaselwanter iwie funkt das nicht, obwohl das das richtige ist laut https://releases.hashicorp.com/terraform/0.12.5/terraform_0.12.5_SHA256SUMS

**Comment by mgrainer** [comment]
@ehaselwanter gleiche problem

**Comment by ehaselwanter** [comment]
@mgrainer no. this is not the same problem: now the converge works but the verify fails. needs to be the same updated check I guess: https://travis-ci.com/infralovers/atlantis-chef/jobs/219554189#L704

File: test/integration/server/inspec/atlantis_spec.rb [hunk 2]
```diff
@@ -11,7 +11,7 @@
   # if needed you may have to install `coreutils` (package name for deb derivitives)
   # root@dokken:/# sha256sum /usr/local/bin/atlantis/atlantis
```
Comment (ehaselwanter) [inline test/integration/server/inspec/atlantis_spec.rb]: @mgrainer did you run this and use that checksum? ... the download checksum most definitely is different from the extracted binary ...

Comment (mgrainer) [reply test/integration/server/inspec/atlantis_spec.rb]: @ehaselwanter yes, tried this at the first attempt..
ok yes you're, download checksum always changing.....

Comment (mgrainer) [reply test/integration/server/inspec/atlantis_spec.rb]: Bildschirmfoto 2019-07-25 um 13 01 49

but the checksum from the binary is neither working

**Pull request merged**